src/components/Ledger/LedgerConnect.stories.js

```diff
@@ -18,30 +18,29 @@ const devices = [
 type Props = {
   visible: any,
   onRequestClose: any,
-  navigation: any,
   defaultDevices: any,
   useUSB: any,
 }
-const LedgerConnectModal = ({visible, onRequestClose, navigation, defaultDevices, useUSB}: Props) => (
-  <Modal visible={visible} onRequestClose={onRequestClose}>
-    {/* $FlowFixMe */}
+const LedgerConnectModal = ({visible, onRequestClose, defaultDevices, useUSB}: Props) => (
+  <Modal visible={visible} onRequestClose={onRequestClose} showCloseIcon>
     <LedgerConnect
-      navigation={navigation}
       onConnectBLE={() => ({})}
       onConnectUSB={() => ({})}
-      onComplete={() => ({})}
       useUSB={useUSB}
       onWaitingMessage={''}
       defaultDevices={defaultDevices}
     />
   </Modal>
 )
 
-const StatefulModal = StorybookModalWrapper(LedgerConnectModal)
-
 storiesOf('Ledger connect', module)
-  .add('Using BLE', ({navigation}) => <StatefulModal navigation={navigation} useUSB={false} />)
-  .add('Using USB', ({navigation}) => <StatefulModal navigation={navigation} useUSB />)
-  .add('BLE with many devices', ({navigation}) => (
-    <StatefulModal navigation={navigation} defaultDevices={devices} useUSB={false} />
+  .addDecorator(withModalProps)
```

Comment:
I think after removing the wrapper, the `showModal` -> visible, is always false, therefore the model is not opening in storybook, now every time we have to click the show model.